.github/workflows/ci.yml

```
@@ -5,19 +5,20 @@ on: pull_request
 jobs:
 
   test:
-    name: Test on ${{ matrix.os }}
+    name: Test on ${{ matrix.os }} with Rust ${{ matrix.rust }}
```

To make the name shorter and less likely to be cut off in certain check views:

```suggestion
    name: Test (+${{ matrix.rust }}) on ${{ matrix.os }}
```